src/openrct2-ui/windows/Ride.cpp

```diff
@@ -6078,7 +6078,7 @@ static void WindowRideGraphsScrollpaint(rct_window* w, rct_drawpixelinfo* dpi, i
                 top = measurement->lateral[x] + 52;
                 bottom = measurement->lateral[x + 1] + 52;
                 intensityThresholdPositive = (RIDE_G_FORCES_RED_LATERAL / 8) + 52;
-                intensityThresholdNegative = -(RIDE_G_FORCES_RED_LATERAL / 8) + 52;
+                intensityThresholdNegative = -(RIDE_G_FORCES_RED_LATERAL / 8) + 47;
```

Comment:
What are these magic numbers for?

Reply:
They determine the threshold in the graph from where the line will be drawn in red. 
 
Even before my change, it was not in sync with the stat screen. I have a park with a Steel Wild Mouse coaster with 2.78 lateral G's, which does not show up in red on the stat screen (as it's <2 .80). However, in the graph, the line is drawn in red at that peak. 

With this change, it now shows up correctly.

Reply:
That is correct.